examples/jsm/interactive/SelectionHelper.js

```diff
@@ -59,6 +59,8 @@ class SelectionHelper {
 		this.renderer.domElement.removeEventListener( 'pointermove', this.onPointerMove );
 		this.renderer.domElement.removeEventListener( 'pointerup', this.onPointerUp );
 
+		this.element.remove(); // in case disposal happens while dragging
```

Comment:
Do you mind changing line 98 from:
```js
this.element.parentElement.removeChild( this.element );
```
to
```
this.element.remove();
```
as well? This style is a lot cleaner.

Reply:
done